war/src/main/less/abstracts/theme.less

```diff
@@ -125,11 +128,17 @@
   --configure-job-help-area-bg-color: var(--medium-translucent);
 
   // Table
-  --even-row-color: #fbfbfb;
-  --bigtable-header-color: var(--very-light-grey);
-  --project-status-border-bottom-color: #ddd;
-  --table-parameters-bg-color--hover: var(--very-light-grey);
-  --table-stripped-bg-color--hover: #e8e8e8;
+  --even-row-color: var(--lightest-grey);
+  --bigtable-border-width: var(--pane-border-width);
+  --bigtable-header-bg: var(--pane-header-bg);
+  --bigtable-header-font-weight: bold; // Does specifying this make sense
+  --bigtable-header-font-size: var(--font-size-md);
+  --bigtable-header-text-color: var(--pane-header-text-color);
+  --bigtable-row-border-color: var(--medium-grey);
+  --bigtable-cell-padding-x: 0.75rem;
+  --bigtable-cell-padding-y: 0.5rem;
+  --table-parameters-bg--hover: var(--light-grey);
+  --table-stripped-bg--hover: var(--light-grey);
```

Comment:
@timja do you agree with this var namespace `--bigtable` instead `--table`?

Reply:
haven't looked as closely as you but why do we need to differentiate tables and bigtables?

Is it because tables are used for layout in many places and big table is actual tabular data?

Reply:
`--table-stripped-bg--hover` is used here at least:
<img width="1409" alt="Captura de pantalla 2020-07-08 a las 10 36 36" src="https://user-images.githubusercontent.com/5738588/86897397-a5464880-c107-11ea-982b-0c0ca529400f.png">

Could be changed. I'm open to feedback.

Reply:
Also, I'm not really sure about inheriting `--pane-*` styles. It feels right but I don't know how well it scales.

Reply:
should this be striped not stripped?

Reply:
Probably yes. It was an existing variable. Changing it.